tools/eslint/config/index.js

```diff
@@ -15,6 +16,12 @@ module.exports = {
   ],
   rules: {
     // ng-cli-compat
+    "jest/max-nested-describe": [
+      "warn",
+      {
+        "max": 4
+      },
+    ],
```

Comment:
Move this under the `custom` section (denoted by the comment). Everything under `ng-cli-compat` and `ng-cli-compat--formatting-add-on` were added to replace deprecated packages and are going to eventually be mass deleted.